packages/loader/driver-definitions/src/storage.ts

```diff
@@ -221,10 +221,31 @@ export interface IDocumentDeltaConnection extends IEventProvider<IDocumentDeltaC
     removeAllListeners(): void;
 }
 
+export enum ServiceCachingPolicy {
+    /**
+     * Driver does not implement any prefetching or caching policy.
+     */
+    None,
+
+    /**
+     * Driver implements prefetching policy, i.e. it prefetches resources from the latest snapshot.
+     */
+    Prefetch,
+}
+
+export interface IDocumentServicePolicies {
+    readonly caching: ServiceCachingPolicy;
```

Comment:
We probably want all policies to be optional with some default fallback? Will make n/n-1 on the api cleaner

Reply:
+1